src/test/java/unit/GcpNotebookControlled.java

```diff
@@ -211,6 +213,12 @@ void overrideLocationAndInstanceId() throws IOException {
     // stored in or accessible via Workspace Manager.
     assertEquals(newName, updatedNotebook.name, "create output matches name");
     assertEquals(newDescription, updatedNotebook.description, "create output matches description");
+    String[] metadatas = newMetadata.split(",");
```

Comment:
when you use a map, the correct format is:
--new-metadata=newFoo=newBar --new-metadata=newFoo2=newBar2. you shouldn't need to do split yourself. See usage example: https://picocli.info/#_maps

Reply:
If you use the `split` argument, you can also do `--new-metadata=foo=bar,baz=quux`, but yes, you don't need to call `String#split`.